src/dataHandler.py

```diff
@@ -1,4 +1,5 @@
 from rocketData import RocketData as rd
+import csv
 class SendData:
     """
     This only deals with sending the rocketdata. RocketData class in src/rocketData.py deals with manipulating the data.  
```

Comment:
can you edit the description of the method in this comment block to say it gets saved to a csv?

Reply:
I have updated it